game/fs.hh

```diff
@@ -99,8 +99,8 @@ Paths const& getPaths();  ///< Get the data file search path
 Paths getThemePaths();  ///< Get the data/theme file search path (includes current and default themes in addition to data folders)
 Paths getPathsConfig(std::string const& confOption);  ///< Return expanded list of paths specified by a path config option
 
-template <>
-struct std::hash<fs::path>
+//template <>
```

Comment:
remove dead template

Reply:
please

Reply:
I have no idea where this dead code is 😆 (fixed)